docs/content/how-to/cdc-ingestion.md

```diff
@@ -274,7 +274,7 @@ flink-sql-connector-kafka-*.jar
 ```
 
 ### Supported Formats
-Flink provides several Kafka CDC formats :canal-json、debezium-json,ogg-json,maxwell-json.
+Flink provides several Kafka CDC formats: canal-json, debezium-json, ogg-jsons, maxwell-json.
```

Comment:
1. `ogg-jsons`? Should it be `ogg-json`
2. How about `Canal, Debezium, Ogg and Maxwell JSONs`

Reply:
good idea